Brightspace/insights-engagement-dashboard
# US121093 [Engagement] Explain when no courses in default view

[US121093](https://rally1.rallydev.com/#/detail/userstory/439372243588?fdp=true): [Engagement] Explain when no courses in default view

LMS PR: https://github.com/Brightspace/lms/pull/3043

### Functional Testing
* Unit tests
* Manual test
  * Navigate to Engagement Dashboard. Confirm that popup windows appears.
    * Confirm that pop-up has courses list if users is enrolled to courses
    * Confirm that pop-up does not have courses list if user is not enrolled to any course within default semesters
    * Confirm that pop-up window does not appear if any further fetch from LMS occurs
### Security Testing - N/A
### Performance Testing - N/A
### Devops - N/A
### UX and docs - N/A


![image](https://user-images.githubusercontent.com/9429561/98008151-e53ee300-1dfc-11eb-9cbb-6c0faca02c3a.png)


FYI @anhill-D2L @MykolaGalian @rohitvinnakota @hyehayes @NicholasHallman
